src/uct/cuda/cuda_ipc/cuda_ipc_md.h

```diff
@@ -1,29 +1,71 @@
 /**
  * Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED.
  * See file LICENSE for terms.
+ * Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
there is different  nvidia licence formats in another file. if you stick this format, make it as follow 

Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED
Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved
See file LICENSE for terms

Reply:
I think as nvidia changes started in this year , i think should just 2018 instead of 2017-2018